prov/gni/src/gnix_msg.c

```diff
@@ -1510,13 +1513,11 @@ static int _gnix_send_req(void *arg)
 	}
 
 	/*
- 	 * if this is a rendezvous message, or GNI_RC_NOT_DONE
- 	 * returned, we might want to generate IRQ at remote
- 	 * peer.
- 	 */
-	if (((status == GNI_RC_SUCCESS) &&
-		(tag == GNIX_SMSG_T_RNDZV_START)) ||
-	    (status == GNI_RC_NOT_DONE))
+	 * if this is a rendezvous message, we want to generate
+	 * IRQ at remote peer.
+	 */
+	if ((status == GNI_RC_SUCCESS) &&
+		(tag == GNIX_SMSG_T_RNDZV_START))
```

Comment:
It appeared that for some cases, we'd get in a loop with status == GNI_RC_NOT_DONE, that lots of interrupts were being generated.  This appeared to bog down the processes a lot since the progress threads would be constantly getting woken up.